components/server/src/server.rs

```diff
@@ -1472,14 +1479,47 @@ where
                     .get_engine_size()
                     .expect("get raft engine size");
 
+                let mut raft_disk_status = disk::DiskUsage::Normal;
+                if separate_raft_path {
+                    let raft_disk_stats = match fs2::statvfs(&raft_path) {
+                        Err(e) => {
+                            error!(
+                                "get disk stat for raft engine failed";
+                                "raft engine path" => raft_path.clone(),
+                                "err" => ?e
+                            );
+                            return;
+                        }
+                        Ok(stats) => stats,
+                    };
+                    let raft_disk_cap = raft_disk_stats.total_space();
+                    let mut raft_disk_available =
+                        raft_disk_cap.checked_sub(raft_size).unwrap_or_default();
+                    raft_disk_available = cmp::min(raft_disk_available, raft_disk_stats.available_space());
+                    raft_disk_status = if raft_disk_available * 100
+                        <= raft_disk_cap * ((100 - raft_engine_almost_full_percent) / 2)
+                    {
+                        disk::DiskUsage::AlreadyFull
+                    } else if raft_disk_available * 100
+                        <= raft_disk_cap * (100 - raft_engine_almost_full_percent)
```

Comment:
Maybe we can just calculate these values just once before the check.

Reply:
These values are meaningful only when `sepearte_raft_path` is true, so calculate them here should be easier to understanding. It may cost very little for very `DEFAULT_STORAGE_STATS_INTERVAL` check.

Reply:
Change `Full Percentage` to `Reserved Space`, no need to calculate these value now. Thanks.